### PR TITLE
simplify language

### PR DESCRIPTION
**Pull request by chinatsu**

### CODE REVIEW

File: docs/naisjob/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: This document describes the properties a Naisjob should have.
 # Naisjob
 A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), just that it only runs once.
```
Comment (x10an14) [inline docs/naisjob/README.md:6]:
```suggestion
A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), except that (by default) it only runs once.
```

File: docs/naisjob/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: This document describes the properties a Naisjob should have.
...
 It is built on Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).
```
Comment (x10an14) [inline docs/naisjob/README.md:7]:
```suggestion
It is built on the Kubernetes [`Job` resource](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).
```

File: docs/naisjob/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: This document describes the properties a Naisjob should have.
...
-In short a Job as opposed to Application will just run a pod one time. 
+Compared to an Application, a Job will only start a pod once.
```
Comment (x10an14) [inline docs/naisjob/README.md:8]:
```suggestion
Compared to an Application, a `Job` will only start a pod once (unless it fails, usual retry-mechanics hold for `Job`s too).
```

File: docs/naisjob/README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ description: This document describes the properties a Naisjob should have.
 # Naisjob
 A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), just that it only runs once.
 It is built on Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).
-In short a Job as opposed to Application will just run a pod one time. 
+Compared to an Application, a Job will only start a pod once. 
 A CronJob runs Jobs on a time-based schedule, as denoted in `.spec.schedule`.
```
Comment (x10an14) [inline docs/naisjob/README.md:9]:
```suggestion
A `CronJob` runs `Job`s on a time-based schedule, as denoted in `.spec.schedule`.
```

**Review by x10an14 — APPROVED** [review]
Minor suggestions =) Only feel strongly about  line 6 tbh.

File: docs/naisjob/README.md (hunk 3 above)
Comment (Kyrremann) [reply docs/naisjob/README.md:8]: Hvis vi skal ha med `unless... ` så syns jeg den burde presis for jobs. Enten lenke til doc, eller ha mer info etter på.

File: docs/naisjob/README.md (hunk 4 above)
Comment (Kyrremann) [reply docs/naisjob/README.md:9]: Fikser du de andre cronjob og job som mangler fnutter?

**Review by Kyrremann — APPROVED** [review]
Kjekt å få rettet på språket. Bare bli enige dere :p så er det bare å merge.

**Comment by chinatsu** [comment]
@x10an14 thoughts? i've expanded upon a few aspects. i should say that i tried to make the documentation reflect the way things currently are, including some of the weird quirks.

File: docs/naisjob/README.md [hunk 5]
```diff
@@ -24,16 +27,21 @@ spec:
...
 ## Shutting down extra containers
-If your Naisjob is running in GCP, or if it is using [Vault sidecar](../naisjob/reference#vaultsidecar) or [Securelogs](../naisjob/reference/#securelogs).
-You need to shut down all containers running in the pod for the job to complete.
+If your Naisjob has any of these traits:
+
+ * [runs in GCP](../clusters/gcp/)
+ * [uses Vault sidecar](reference/#vaultsidecar)
+ * [uses Securelogs](reference/#securelogs)
+
+You will need to shut down all containers running in the pod for the `Job` to complete.
```
Comment (x10an14) [inline docs/naisjob/README.md:44]: Here I feel as if lines 30-31 (in new version) are repeated (almost verbatim) in more verbose (and easier to read) form in lines 38-44.

Here's my suggested edit:
```suggestion
    Make sure to read the caveat [about shutting down all containers](#shutting-down-extra-containers) for your Naisjob to successfully complete.

## Applying your Naisjob to Kubernetes

You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).

## Shutting down extra containers

If your Naisjob has any of these traits:

     * [runs in GCP](../clusters/gcp/)
     * [uses Vault sidecar](reference/#vaultsidecar)
     * [uses Securelogs](reference/#securelogs)

You will need to shut down all containers running in the pod for the `Job` to complete.
```

**Comment by x10an14** [comment]
My two cents: 

Merge after reviewing [my feedback](https://github.com/nais/doc/pull/198#discussion_r649091413) independently of whether you agree with the feedback or not =)
